### DISCUSSION TITLE
Use of Vorta to control remote backups?

### DISCUSSION THREAD

**Discussion by akostadinov** [borgbase/vorta#2177] [FAQ] [answered]
Hello, I can't find information whether it is possible to use Vorta remotely.

My use case is that I have a headless server that I want to backup to another server with Borg but control this from my workstation which is neither client nor server for the backup.

So is it possible to control another computer's backup by running Vorta on my workstation?

**Comment by m3nu** [comment, marked as answer]
Not a use case we focus on, but you could use the Vorta Docker image for this purpose. I believe it comes with a VNC server.

https://github.com/borgbase/vorta-docker

**Reply by akostadinov** [reply]
Thank you for the information. It is unfortunate though :cry:

**Reply by m3nu** [reply]
Using the Docker image should fully cover this use case. I.e. you connect via VNC to the remote server to control Vorta there.

If this is a headless server, you may want to consider using Borgmatic or Borg directly.

**Reply by akostadinov** [reply]
yeah, I'm looking at the clis. I thought that for once I may go easy and use a GUI but that's life. Running vorta from a container is not much different than running directly on server (provided it is already in the package manager) because the container will need root access anyway. And in my setup I only run rootless. So it complicates matters more than I'm ready to accept at this point.

But thank you for listing the options so I know what is possible and what isn't.
